src/types/Misc.ts

```diff
@@ -0,0 +1,4 @@
+export enum EnvType {
```

Comment:
Misc isn't a great name either. Types wasn't good either though haha

Reply:
How about `Common`? Sounds slightly better than `Misc`, I think. :thinking:

Reply:
Sure that's better, I guess we'd get in the weeds of a bunch of variable files if they all are named specific